driver.sh

```diff
@@ -332,26 +339,38 @@ check_objcopy_strip_version() {
     done
 }
 mako_reactor() {
-    # https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/Documentation/kbuild/kbuild.txt
-    time \
-        KBUILD_BUILD_TIMESTAMP="Thu Jan  1 00:00:00 UTC 1970" \
-        KBUILD_BUILD_USER=driver \
-        KBUILD_BUILD_HOST=clangbuiltlinux \
-        make -j"${jobs:-$(nproc)}" \
-        AR="${AR}" \
-        CC="${CC}" \
-        HOSTCC="${CC}" \
-        HOSTLD="${HOSTLD:-ld}" \
-        KCFLAGS="-Wno-implicit-fallthrough" \
-        LD="${LD}" \
-        LLVM_IAS="${LLVM_IAS}" \
-        NM="${NM}" \
-        OBJCOPY="${OBJCOPY}" \
-        OBJDUMP="${OBJDUMP}" \
-        OBJSIZE="${OBJSIZE}" \
-        READELF="${READELF}" \
-        STRIP="${STRIP}" \
-        "${@}"
+    if [[ ${LLVM} -eq "1" ]]; then
```

Comment:
Will probably need something like `${LLVM:-0}` due to `set -u`.

Reply:
yep, done